PurchasesCoreSwift/Networking/HTTPRequest.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  HTTPRequest.swift
+//  PurchasesCoreSwift
+//
+//  Created by Juanpe Catalán on 8/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+public typealias HTTPClientResponseHandler = (Int, [String: Any]?, NSError?) -> Void
+
+// TODO(post migration): Change back to internal
+@objc(RCHTTPRequest)
+public class HTTPRequest: NSObject, NSCopying {
+    @objc public var httpMethod: String
```

Comment:
These all can be `let` 🎉

Reply:
Firstly, all these were `let`, but when I checked the previous code, I realized that the properties weren't `readonly`, so I used `var` instead. TBH, I agree with you, let me change them 👍🏼

Reply:
Our approach has been to basically fix small things like that where it doesn't impact public API 😄
Public API must stay the same, if we can.